ihatemoney/templates/forms.html

```diff
@@ -69,12 +69,19 @@
 
     {% include "display_errors.html" %}
     {{ form.hidden_tag() }}
+    <!-- Only display private code if not home page -->
     {% if not home %}
-    {{ input(form.id) }}
+    {{ input(form.id, placeholder="Unique ID used to login") }}
+    {% endif %}
+    <!-- Fields that are always shown -->
+    {{ input(form.name, placeholder="Example: Trip to Paris") }}
+    {{ input(form.password, placeholder="Password to access project") }}
+    {{ input(form.contact_email, placeholder="Your email") }}
+    <!-- Only display member name and default currency if not home page -->
```

Comment:
I feel this is weird. My experience with the project creation page (not on the home) is that it's shown to users only if there is an error when validating the form from the home.

I don't think we should add some specific fields here in this case, and not in the home. From the UX perspective it feels weird : we should have it on both locations or nowhere.